kbfsfuse/main.go

```diff
@@ -30,13 +30,13 @@ const usageFormatStr = `Usage:
 
 To run against remote KBFS servers:
   kbfsfuse
-    [-runtime-dir=path/to/dir] [-label=label] [-mount-type=force]
+    [-runtime-dir=path/to/dir] [-label=label] [-mount-type=` + defaultMountType + `]
```

Comment:
Should this be something like "-mount-type=[default|force|none]"?

Reply:
sure; will update!